templates/patterns/page_header/pattern-page-header--variant-background-image.html.twig

```diff
@@ -0,0 +1,20 @@
+{#
+/**
+ * @file
+ * Background image variant for the page header component.
+ *
+ * Have difference only for "core" ecl branding.
+ */
+#}
+
+{% include '@oe_theme/patterns/page_header/pattern-page-header.html.twig' with {
+  'title': title,
+  'introduction': introduction,
+  'metas': metas,
+  'background_image': ecl_branding == 'core' and background_image.src is not empty ? true : false,
```

Comment:
This `ecl_branding == 'core' and background_image.src is not empty` already returns a bool, no need for  `? true : false,`

Reply:
fixed